src/test/rustdoc-gui/basic.goml

```diff
@@ -1,4 +1,4 @@
 goto: file://|DOC_PATH|/index.html
-assert: ("#functions")
+assert: ("#functi")
```

Comment:
Why did this change?

Reply:
It's part of my second commit "TO BE REMOVED". I want to be sure the CI is failing because this test failed. :)